lib/tabular_data/container.rb

```diff
@@ -25,6 +26,12 @@ def rows
       results
     end
 
+    # rubocop:disable all
+    def rows=(new_rows)
+      @rows = new_rows
+    end
+    # rubocop:enable all
```

Comment:
I tried that first, and things blew up, I think because the assignment syntax was not expected. I.e. this:
```
container.rows( new_rows )  # ok
container.rows = new_rows  # not ok
```